### PR TITLE
SONARHTML-138 S5254 language_attributes() call should be a valid language attribute supplier

### PR DESCRIPTION
**Pull request by thomas-serre-sonarsource**

### CODE REVIEW

File: sonar-html-plugin/src/test/resources/checks/LangAttributeCheck.html [hunk 1]
```diff
@@ -13,3 +13,5 @@
 <html lang="en" xml:lang="en"> <!-- Compliant -->
   <head></head>
 </html>
+
+<html  <?php language_attributes(); ?>> <head></head> </html> <!-- Compliant -->
```
Comment (christophe-zurn-sonarsource) [inline sonar-html-plugin/src/test/resources/checks/LangAttributeCheck.html:17]: _minor_ this is an internal convention to end files with newlines

File: sonar-html-plugin/src/main/java/org/sonar/plugins/html/checks/sonar/LangAttributeCheck.java [hunk 2]
```diff
@@ -36,6 +37,21 @@ private static boolean isHtmlTag(TagNode node) {
...
+   * Using WordPress, HTML attributes can be set using the php function `language_attributes`
+   */
+  private static boolean hasLangAttributeFromWordPress(TagNode node) {
+    for (Attribute attribute : node.getAttributes()) {
+      String attributeName = attribute.getName();
+      if (attributeName.contains("?php") && attributeName.contains("language_attributes")) {
+        return true;
+      }
+    }
+    return false;
```
Comment (christophe-zurn-sonarsource) [inline sonar-html-plugin/src/main/java/org/sonar/plugins/html/checks/sonar/LangAttributeCheck.java:55]: This is a matter of preference of implementation, but it's also possible to use Java 8 streams for this logic as follows:
```suggestion
    return node.getAttributes().stream()
      .map(Attribute::getName)
      .anyMatch(attributeName -> attributeName.contains("?php") && attributeName.contains("language_attributes"));
```
Up to you to keep as is or update :)

**Review by christophe-zurn-sonarsource — APPROVED** [review]
LGTM !
Nicely done :)

File: sonar-html-plugin/src/main/java/org/sonar/plugins/html/checks/sonar/LangAttributeCheck.java (hunk 2 above)
Comment (thomas-serre-sonarsource) [reply sonar-html-plugin/src/main/java/org/sonar/plugins/html/checks/sonar/LangAttributeCheck.java]: I chose the stream version ;) Thanks for the java refresher :)

**Pull request merged**